azure-db/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG PG_IMAGE=postgres:11-alpine
+
+FROM ${PG_IMAGE}
+
+RUN apk add --no-cache python python-dev build-base gcc wget git py-pip py-setuptools libffi-dev openssl-dev
+RUN pip install azure-cli
+RUN apk del python-dev build-base gcc wget py-pip libffi-dev openssl-dev
```

Comment:
Can merge the runs for better docker caching

Reply:
merge with `&&`

Reply:
?

Reply:
do the fix :)